### PR TITLE
Added methods to get and update a user to the user controller and routes for these methods

### PR DESCRIPTION
**Pull request by tsimian**
issue: #152

I also added...
* nodemon as a dependency, this way we won't have to keep restarting the server after making changes on the back end
* a new script to package.json ```npm run dev``` for running the server with nodemon
* comments labelling user methods

### CODE REVIEW

File: server/src/controller/userController.js [hunk 1]
```diff
@@ -41,4 +44,44 @@ const userSignUp = async (req, res) => {
...
+      email,
+      postalCode,
+      roles,
+    });
+  } catch (err) {
+    res.status(400).json({ error: err.message });
+  }
+};
+
+export { userLogin, userSignUp, getMe, updateMe };
```
Comment (lugenx) [inline server/src/controller/userController.js:87]: Nice work! Would it make sense to include `deleteMe` as well?

**Review by lugenx — CHANGES_REQUESTED** [review]
Good job! A couple of comments for you to address, please.

File: server/src/controller/userController.js (hunk 1 above)
Comment (tsimian) [reply server/src/controller/userController.js:87]: That's a great idea! I'll add that now.

**Comment by lugenx** [comment]
This looks good to me, thank you!

**Review by lugenx — APPROVED** [review]

**Pull request merged**